server/services/drafts/draftReportService.ts

```diff
@@ -119,6 +123,32 @@ export default class DraftReportService {
     return this.handleAddStaff(token, user, bookingId, [userToAdd])
   }
 
+  async getPotentialDuplicates(
+    bookingId: number,
+    incidentDate: Moment,
+    token: string
+  ): Promise<Partial<DuplicateReport[] | []>> {
+    const startDate = moment(incidentDate).startOf('day')
+    const endDate = moment(incidentDate).endOf('day')
+    const reports = await this.draftReportClient.getDuplicateReports(bookingId, [startDate, endDate])
+    return Promise.all(
+      reports
+        .filter(report => report.status !== ReportStatus.IN_PROGRESS.value)
```

Comment:
think we talked about pushing this check into the query.. could do as part of your follow up?

Reply:
yes